javaTask/src/banking/BankAccount.java

```diff
@@ -0,0 +1,50 @@
+package banking;
+
+public class BankAccount {
+    private int accountNumber;
+    private String accountHolderName;
+    private int balance;
+    private int addMoney;
+
+
+    public void setAccountNumber(int accountNumber) {
+        this.accountNumber = accountNumber;
+    }
+
+    public void setBalance(int balance) {
+        this.balance = balance;
+    }
+
+    public int getAccountNumber() {
+        return accountNumber;
+    }
+
+    public String getAccountHolderName() {
+        return accountHolderName;
+    }
+
+    public void setAccountHolderName(String accountHolderName) {
+        this.accountHolderName = accountHolderName;
+    }
+
+    public int getBalance() {
+        return balance;
+    }
+
+    public void depositingMoney(int addMoney) {
```

Comment:
I recommend renaming this parameter to money/amount/addedMoney instead of addMoney, as `addMoney` matches the methods naming conventions not variables or data members.